tests/components/switcher_kis/conftest.py

```diff
@@ -0,0 +1,108 @@
+"""Common fixtures and objects for the Switcher integration tests."""
+
+from asyncio import Queue
+from datetime import datetime
+from typing import Any, Generator, Optional
+
+from asynctest import CoroutineMock, patch
+from pytest import fixture
+
+from .consts import (
+    DUMMY_AUTO_OFF_SET, DUMMY_DEVICE_ID, DUMMY_DEVICE_NAME,
+    DUMMY_DEVICE_STATE, DUMMY_ELECTRIC_CURRENT, DUMMY_IP_ADDRESS,
+    DUMMY_MAC_ADDRESS, DUMMY_PHONE_ID, DUMMY_POWER_CONSUMPTION,
+    DUMMY_REMAINING_TIME)
+
+
+@patch('aioswitcher.devices.SwitcherV2Device')
+class MockSwitcherV2Device:
+    """Class for mocking the aioswitcher.devices.SwitcherV2Device object."""
+
+    def __init__(self) -> None:
+        """Initialize the object."""
+        self._last_state_change = datetime.now()
+
+    @property
+    def device_id(self) -> str:
+        """Return the device id."""
+        return DUMMY_DEVICE_ID
+
+    @property
+    def ip_addr(self) -> str:
+        """Return the ip address."""
+        return DUMMY_IP_ADDRESS
+
+    @property
+    def mac_addr(self) -> str:
+        """Return the mac address."""
+        return DUMMY_MAC_ADDRESS
+
+    @property
+    def name(self) -> str:
+        """Return the device name."""
+        return DUMMY_DEVICE_NAME
+
+    @property
+    def state(self) -> str:
+        """Return the device state."""
+        return DUMMY_DEVICE_STATE
+
+    @property
+    def remaining_time(self) -> Optional[str]:
+        """Return the time left to auto-off."""
+        return DUMMY_REMAINING_TIME
+
+    @property
+    def auto_off_set(self) -> str:
+        """Return the auto-off configuration value."""
+        return DUMMY_AUTO_OFF_SET
+
+    @property
+    def power_consumption(self) -> int:
+        """Return the power consumption in watts."""
+        return DUMMY_POWER_CONSUMPTION
+
+    @property
+    def electric_current(self) -> float:
+        """Return the power consumption in amps."""
+        return DUMMY_ELECTRIC_CURRENT
+
+    @property
+    def phone_id(self) -> str:
+        """Return the phone id."""
+        return DUMMY_PHONE_ID
+
+    @property
+    def last_data_update(self) -> datetime:
+        """Return the timestamp of the last update."""
+        return datetime.now()
+
+    @property
+    def last_state_change(self) -> datetime:
+        """Return the timestamp of the state change."""
+        return self._last_state_change
+
+
+@fixture
```

Comment:
Move the scope parameter here and also give it a name to avoid outer scope redefinition warnings: `@fixture(name='mock_bridge', scope='module')`

Reply:
The scope is not needed at all, I'm not using it.
I just accidentally pasted it in the wrong (and harmless) parenthesis of the function instead of the decorator. That's my bad :-)
But you are right about the name parameter.
I'll add it.

Reply:
Fixed in this [commit](https://github.com/home-assistant/home-assistant/pull/22325/commits/6b2c94286e9db9b7f0e21baf9d3c9357e37a5fba).